soda/src/main/java/it/okkam/opendata/models/soda/SodaClassification.java

```diff
@@ -50,4 +51,12 @@ public void setDomain_metadata(Map<String, String>[] domain_metadata) {
     this.domain_metadata = domain_metadata;
   }
 
+  public Map<String, String>[] getDomain_private_metadata() {
+    return domain_private_metadata;
+  }
+
+  public void setDomain_private_metadata(Map<String, String>[] domain_private_metadata) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00117)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://dev.okkam.it/sonar/coding_rules#rule_key=squid%3AS00100)